docs/modules/ROOT/partials/install/bootstrap-nodes.adoc

```diff
@@ -132,6 +163,30 @@ terraform apply
 popd
 ----
 
+. Review and merge the LB hieradata MR (listed in Terraform output `hieradata_mr`) and run Puppet on the LBs after the deploy job has completed
```

Comment:
This won't be sufficient for cloudscale anymore, since this also ensures that the ingress backends are set to the infra node IPs. This isn't done anymore for cloudscale since we set `infra_count=0` and we'll need a replacement step after scaling up the infra nodes which sets their IPs in the hieradata so that the ingress HAProxy config has backends.